.reek

```diff
@@ -21,6 +21,9 @@ UtilityFunction:
    enabled: false
   FeatureEnvy:
    enabled: false
+"app/controllers":
+  InstanceVariableAssumption:
```

Comment:
this check makes no sense, instance variables are initialised inside methods in Rails